package.json

```diff
@@ -71,7 +71,7 @@
       "prettier --write --prose-wrap always"
     ],
     "*.svg": [
-      "svgo --disable=removeViewBox --pretty"
+      "svgo --pretty"
```

Comment:
Just checking in with @ludwiczakpawel for this setting.

Reply:
In svgo.config.js